### ISSUE TITLE
Design "Add Your Club" workflow

### ISSUE THREAD

**Issue by hannahkane** [mozilla/teach.webmaker.org#70]
### RACI
* Phase: Design
* Owner: @hannahkane 
* Decision Maker: @hannahkane 
* Design Lead: @vazquez
* Dev Lead: NA
* Quality: @davidascher 

### "Add Your Club" workflow

Entry points: Two "Add Your Club" buttons on the Clubs page

The design has already been started: https://redpen.io/efdaacb2e7893b8da3/10

### Other workflows

* User delete: the user needs to be able to delete their Club from the map, should require a confirmation step
* Staff delete: users with staff privs need to be able to delete Clubs from the map, should require confirmation

**Comment by hannahkane** [comment]
"Add Your Club" workflow is here: https://redpen.io/wc356809d269ed0827. There are a couple more  redpen comments to incorporate.

**Comment by vazquez** [comment]
Design Prototype: http://invis.io/HM2CQUPD2

@hannahkane @iamjessklein

**Comment by hannahkane** [comment]
@vazquez - that link goes to the site map for me. I think this is a correct updated link: https://projects.invisionapp.com/d/main#/console/2977378/66113218/preview

It looks great to me. I think we need to think through the authentication process, though, for the case where the user is not yet logged in when they click on "Add your club."

**Comment by vazquez** [comment]
@hannahkane - I'm blocked by this authentication, since I'm not quite sure how the new Login will work. I can design with the current implementation of Login, but I think that will change as the workflow is being coded.

**Comment by hannahkane** [comment]
ah, of course. OK, this ticket is officially dependent on #227 :)

**Comment by hannahkane** [comment]
Closing this as it's a duplicate of https://github.com/mozilla/id.webmaker.org/issues/51
